### PR TITLE
Added base code of conduct

### PR DESCRIPTION
**Pull request by frankcorneliusmartin**
Community leaders == Community Admins in our case?

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (bartvanb) [inline CODE_OF_CONDUCT.md:64]: well what should we put here? Point them to discord and define there who the admins are?

File: README.md [hunk 2]
```diff
@@ -41,7 +43,7 @@ _A High level overview of the vantage6 infrastructure. Vantage6 has both a clien
 
 ### Requirements
 
-The **vantage6** infrastructure is delivered in Docker images. To run these images, you need to have [Docker](https://docs.docker.com/get-docker/) installed. To install the latest version of the vantage6 CLI, you need to have [Python](https://www.python.org/downloads/), we recommend using an environment manager like [mini-conda](https://docs.conda.io/en/latest/miniconda.html).
+The **vantage6** infrastructure is delivered in Docker images. To run these images, you need to have [Docker (docs.docker.com)](https://docs.docker.com/get-docker/) installed. To install the latest version of the vantage6 CLI, you need to have [Python (python.org)](https://www.python.org/downloads/), we recommend using an environment manager like [mini-conda (docs.conda.io)](https://docs.conda.io/en/latest/miniconda.html).
```
Comment (bartvanb) [inline README.md:46]: I guess you added these domain names to the link to make them more trustworthy but personally I don't like this as it disrupts the reading flow and also there are no guarantees that you actually link to those domains so it's a false sense of security. 
All in all for me, the added value would only outweigh the cons if we expect a lot of people want to click these links, which at least for the ones above I don't think will happen

Comment (bartvanb) [reply README.md:46]: Also line length is too long, best to spread over a few lines.

**Review by bartvanb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (frankcorneliusmartin) [reply README.md:46]: I added them to make clear they are brought to an external website. Not for trust, never trust link names...

**Review by dsmits — APPROVED** [review]
LGTM

**Review by CunliangGeng — APPROVED** [review]
LGTM too. Nice to add the code of conduct!

File: README.md (hunk 2 above)
Comment (bartvanb) [reply README.md:46]: Sure, then not for trust but to show where they are leading. To me, making these links more clear is still far less important than the adverse effect it has on people being able to read through this paragraph easily. Note that this is one of the first places where people looking into vantage6 read about it, so I think it is important to write clearly here.

Comment (frankcorneliusmartin) [reply README.md]: Ok I yield. I also removed all other link targets from all other links in this readme.

**Review by bartvanb — APPROVED** [review]
Muy bien

**Pull request merged**